hbase-common/src/main/resources/hbase-default.xml

```diff
@@ -1901,4 +1901,24 @@ possible configurations would overwhelm and obscure the important.
       automatically deleted until it is manually deleted
     </description>
   </property>
+  <property>
+    <name>hbase.master.regions.recovery.interval</name>
```

Comment:
Name it like hbase.master.regions.recovery.check.interval (?)  Is that a better name?